cdap-ui/app/features/adapters/adapters.less

```diff
@@ -14,42 +14,523 @@
  * the License.
  */
 
-@import "../../styles/variables.less";
+@import '../../styles/variables.less';
 @import "../../../bower_components/bootstrap/less/mixins.less";
-@import "../../styles/themes/cdap/mixins.less";
+@import '../../styles/themes/cdap/mixins.less';
+
 body.theme-cdap {
-  &.state-adapters-drafts, .state-adapters.create {
-    [apps-drafts-list] {
-      [class*="col-"] {
-        padding-left: 5px;
-        padding-right: 5px;
+  &.state-adapters {
+    margin-bottom: 50px;
+
+    main.container {
+      width: 100%;
+      padding-right: 0;
+      padding-left: 0;
+
+      & > div {
+        margin-bottom: 0
+      }
+    }
+
+    i.fa-exclamation-circle {
+      margin-left: 3px;
+      outline: none;
+    }
+    .fa-plus-square { color: @cdap-header; }
+    .fa-info-circle {
+      color: @brand-primary;
+    }
+    fieldset.view-plugin-json {
+      textarea {
+        resize: none;
+        height: 60vh;
+      }
+    }
+  }
+
+  &.state-adapters-detail, &.state-adapters-create {
+    .btn {
+      &.btn-orange, &.btn-grey {
+        color: white;
+        padding: 9px 20px;
+      }
+      &.btn-orange {
+        background-color: @cdap-orange;
+      }
+
+      &.btn-grey {
+        background-color: #ccc;
       }
-      .drafts-list-container {
-        margin-left: 10px;
-        margin-right: 10px;
+    }
+  }
+
+  &.state-adapters-detail {
+    my-dag {
+      width: 100%;
+      height: 40vh;
+    }
+    my-program-history {
+      ul.pagination {
+        margin-right: 15px;
       }
     }
+  }
+
+  &.state-adapters-create {
+    background-color: #e2e3e5;
+    // Adjust margins for the grid background
+    margin-bottom: 52px;
     main.container {
       background-color: transparent;
-      background-image: linear-gradient(@body-bg, @body-bg);
+      background-image: none;
       margin-top: 60px;
       > div {
         margin-bottom: 0;
       }
     }
-  }
-  &.state-adapters {
 
-    fieldset.view-plugin-json {
-      textarea {
-        resize: none;
-        height: 60vh;
+    .modal .modal-header {
```

Comment:
Modal classes are (still) there for Export, which is the only modal being used in Hydrator at this point. I plan to remove it in the PR that consolidates modal styling.